documentation/Installation.md

```diff
@@ -2623,10 +2623,10 @@ The following settings are supported:
     <td>By enabling ready, an HTTP endpoint on port 8181 returns 200 OK when all plugins that are able to signal readiness have done so. If some are not ready, the endpoint still returns a 503 with the body containing the list of plugins that are not ready. Once a plugin has signaled that it is ready, it is not queried again.</td>
   </tr>
   <tr>
-    <td>prometheus</td>
+    <td>Prometheus</td>
```

Comment:
Hi @shmo1218 
This is a key. It might be case sensitive.

Reply:
Hi, @ilia1243,

Thank you for pointing it out. I have rollbacked the change.